src/redux/reducers/articleReducer.js

```diff
@@ -0,0 +1,8 @@
+export default function articleReducer(state = [], action) {
+  switch (action.type) {
+    case "CREATE_ARTICLE":
```

Comment:
could you please change action types from string to constant variables? maybe you can create a file for all action types variables

Reply:
Roger that @j4l13n i'll get right on it.